### ISSUE TITLE
`@` import paths require upstream dependencies

### ISSUE THREAD

**Issue by noslouch** [cibernox/ember-basic-dropdown#327]
the new import paths in #315 are incompatible with upstream addons if they don't haven't updated both `ember-cli-babel` and `ember-cli-htmlbars-inline-precompile`. might want to make a note of that.

**Comment by cibernox** [comment]
@noslouch what do you mean by upstream addons? In theory every addon should be transpiled with the version of `ember-cli-babel` specified in its package.json, so this should work even if the consumer apps is in an old version of babel.

**Comment by noslouch** [comment]
The issue came with importing `clickTrigger`. Older versions of babel don't like [this syntax](https://github.com/cibernox/ember-basic-dropdown/blob/master/test-support/helpers/ember-basic-dropdown.js#L1) and would die when it gets to that line.

**Comment by cibernox** [comment]
Oh yes. Addon code is transpiled with the version of babel in the addon, but `test-support` is merged into the app tree, so it's not true for the files on that folder... Good point.
I'll check if there is something I can do on my end.

**Comment by noslouch** [comment]
FWIW I was able to update `ember-cli-babel` and `ember-cli-htmlbars-inline-precompile` all the way up my dependency tree (`ember-basic-dropdown` is a dep of an addon which is used in my main app) without negative effects.

a bolded note in the readme about being sure your app is using the latest version of those packages might be enough.

**Comment by cibernox** [comment]
@noslouch #328 This PR should make this thing work regardless of the version of ember-cli-babel in your app. If I publish a beta version could you verify that it works?

**Comment by noslouch** [comment]
yeah!

**Comment by cibernox** [comment]
@noslouch try version v0.33.1-beta.1
With this it should work even if you force your app to stay in babel `~6.4.0`.

**Comment by cibernox** [comment]
@noslouch Any luck?

**Comment by noslouch** [comment]
a little trouble with the install. looks like `addon-test-helpers.js` isn't there when I install the beta. investigating.

**Comment by cibernox** [comment]
@noslouch I published beta.0 which was busted and then I published beta.1 which should work. Are you on that?

**Comment by noslouch** [comment]
yeah I'm using beta.1, but even in a new ember project, I'm not seeing the addon-test-helpers.js file
```
> $ npm install ember-basic-dropdown@0.33.1-beta.1
+ ember-basic-dropdown@0.33.1-beta.1
added 7 packages in 6.264s

> $ ls node_modules/ember-basic-dropdown
CHANGELOG.md LICENSE.md   README.md    addon        app          index.js     node_modules package.json test-support vendor
```

**Comment by cibernox** [comment]
This has been fixed in #328
